core/src/main/java/io/questdb/griffin/engine/functions/eq/EqByteFunctionFactory.java

```diff
@@ -61,6 +61,9 @@ public Func(Function left, Function right) {
 
         @Override
         public boolean getBool(Record rec) {
+            if(left.isNullConstant() || right.isNullConstant()) {
+                return negated || false;
```

Comment:
ditto 

Table of Truth:

| negated | result |
|---------|-------|
| false      | false   |
| true       | true     |

the `|| false` is thus redundant.